src/Game.java

```diff
@@ -0,0 +1,64 @@
+import java.util.*;
+
+public class Game {
+    private List<String> words = new ArrayList<>();
+    Random random = new Random();
+    WordsDatabase database = new WordsDatabase(words, random);
+    public final int exit = 0;
+    public final int play = 1;
+    private int opition = 1;
+    Scanner sc = new Scanner(System.in);
+
+    public void hangman() {
+        try {
+            while (opition != exit) {
+
+
+                switch (opition) {
+                    case exit:
+                        System.out.println("Koniec gry.");
+                        sc.close();
+                        break;
+                    case play:
+                        game();
+                        break;
+                    default:
+                        System.out.println("Nie ma takiej opcji.");
+                }
+                System.out.println("Dostępne opcję:");
+                System.out.println("0 - wyjście z programu.");
+                System.out.println("1 - graj dalej");
+                opition = sc.nextInt();
+                sc.nextLine();
+
+            }
+        } catch (InputMismatchException e) {
+            System.out.println("Nie mozna podawać znaków niezgodnych z poleceniem.");
+        }
+    }
+
+    private void game() {
+        Hangman hangman = new Hangman(new ArrayList<>(), new ArrayList<>(), new ArrayList<>());
```

Comment:
Three empty arrays as parameters. What the hell is this? Why cannot we create them in the Hangman constructor instead?